chapter_convolutional-modern/vgg.md

```diff
@@ -12,8 +12,8 @@
 
 经典卷积神经网络的基本组成部分是下面的这个序列：
 1. 带填充以保持分辨率的卷积层；
-1. 非线性激活函数，如ReLU；
-1. 池化层，如最大池化层。
+2. 非线性激活函数，如ReLU；
+3. 池化层，如最大池化层。
```

Comment:
Please revert back to the original. `.md` file will auto-increase the number sequence when it's rendering.